protocol/rpcprovider/rpcprovider_server.go

```diff
@@ -113,7 +113,7 @@ func (rpcps *RPCProviderServer) ServeRPCRequests(
 	relaysMonitor *metrics.RelaysMonitor,
 	providerNodeSubscriptionManager *chainlib.ProviderNodeSubscriptionManager,
 	staticProvider bool,
-	providerLoadManager *ProviderLoadManager,
+	relayLoadLimit uint64,
```

Comment:
you do need this pointer (revert change)

Reply:
Fixed in [9eabb5a](https://github.com/lavanet/lava/pull/1720/commits/9eabb5ac988c7c12d533a62a1f2bfa88eeb9425a)